test/src/gateways/postgres/plugin.test.js

```diff
@@ -1,10 +1,11 @@
 import pg from "pg";
-import PostgresGateway from "../../../../src/gateways/postgres/plugin";
+import PostgresGateway from "../../../../src/gateways/postgres/index";
```

Comment:
**Annotation:** Should not directly import from the plugin file. Changed to import from index instead.